Rediscovery design

detig-iii opened the pull request:
Adds a new design "Re:Discovery", a new imagepack "Re:Discovery" and a web app manifest.
The imagepack so far replaces the images of a few ships with more to come later.

glaubinix commented:
Selecting the new design currently just renders a blank page with a background image. I believe you need to set up the design in `layout/game.html.twig` and adjust the `design/rediscovery.html.twig` to be similar to other designs, not extend the base layout and not use the block.

detig-iii commented:
I have implemented the suggested changes to conform to the older version of the design integration and successfully tested it on version 3.6 stable. I have also added a minor improvement to the planet circle view.
